pkgs/by-name/op/open-vm-tools/package.nix

```diff
@@ -51,13 +51,13 @@ in
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "open-vm-tools";
-  version = "12.5.0";
+  version = "12.5.2";
 
   src = fetchFromGitHub {
     owner = "vmware";
     repo = "open-vm-tools";
-    rev = "stable-${finalAttrs.version}";
-    hash = "sha256-pjMXhVN4xdmPCk1Aeb83VZjDJ1t1mb9wryC6h3O+Qvc=";
+    tag = finalAttrs.version;
```

Comment:
I don't see a `12.5.2` in the repo. Shouldn't this be prefixed with `stable-`?

Reply:
It was not when I created the PR. It appears they have dropped the 12.5.2 tag they had published in favor of a `stable-12.5.2` tag...